runtime/isp_run_app.py

```diff
@@ -45,6 +45,9 @@ def main():
     parser.add_argument("-r", "--runtime", type=str, default="bare", help='''
     Currently supported: frtos, sel4, bare (bare metal) (default), stock_frtos, stock_sel4, stock_bare
     ''')
+    parser.add_argument("--arch", type=str, default="rv32", help='''
```

Comment:
Is `-a` being used for something else? We should add it as a shortcut

Reply:
Ok.